code/_globalvars/lists/misc.dm

```diff
@@ -20,6 +20,8 @@ var/list/restricted_camera_networks = list( //Those networks can only be accesse
 	"Thunderdome",
 	"UO45",
 	"UO45R",
+	"UO71",
+	"UO71R",
```

Comment:
The initial map was copied from UO45, which apparently had two. I did not go back and change them to be one network.